### PR TITLE
Feature/join unimind section

### PR DESCRIPTION
**Pull request by TomaszKotowski**

### CODE REVIEW

**Review by 99mac99 — DISMISSED** [review]

File: theme/theme.d.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+declare module "@mui/material/styles" {
+  interface Theme {}
+  // allow configuration using `createTheme`
```
Comment (adiusz) [inline theme/theme.d.ts]: we don't need that comment :P

File: theme/theme.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { createTheme, Theme, ThemeOptions } from "@mui/material";
+
+// A custom theme for this app
```
Comment (adiusz) [inline theme/theme.ts]: and this also

**Review by adiusz — CHANGES_REQUESTED** [review]

File: components/About/SingleCard.styled.tsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    linear-gradient(
+      180deg,
+      rgba(255, 255, 255, 0.05) 0%,
+      rgba(255, 255, 255, 0) 100%
+    );
+  border-radius: 10px;
+  height: 100%;
+  padding-bottom: ${({ theme }) =>
+    theme.breakpoints.down("md") ? "3.5rem" : ""};
+  /* padding-bottom: isMobile ? "3.5rem" : "", */
```
Comment (adiusz) [inline components/About/SingleCard.styled.tsx]: another comment

File: components/About/About.tsx [hunk 4]
```diff
@@ -5,6 +5,24 @@ import { Content } from "../../layout/Content";
 
 import { SingleCard } from "./SingleCard";
 
+const CARDS_TITLES = {
```
Comment (adiusz) [inline components/About/About.tsx]: why don't create array of objects wuth title and description, keep it in seperate file (i like to have a data folder if not external solution like CMS/DB is used — but its not that important in this case I guess) and then use a map method to render all of the items that looks the same but have just different texts?

File: components/About/SingleCard.styled.tsx [hunk 5]
```diff
@@ -14,7 +14,8 @@ export const StyledSingleCard = styled(Paper)`
       rgba(255, 255, 255, 0) 100%
     );
   border-radius: 10px;
-  height: 100%;
-  padding-bottom: ${({ theme }) =>
-    theme.breakpoints.down("md") ? "3.5rem" : ""};
+  /* height: 100%; */
+  ${({ theme }) => theme.breakpoints.up("md")} {
+    /* padding-bottom: 0; */
```
Comment (adiusz) [inline components/About/SingleCard.styled.tsx]: commnetzzzzz

File: components/DrivingSection/DrivingSection.tsx [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+export const DrivingSection = ({
+  title,
+  children,
+  icon,
+}: DrivingSectionType) => {
+  const theme = useTheme();
+
+  return (
+    <Grid container direction="column">
+      {/* Icon */}
```
Comment (adiusz) [inline components/DrivingSection/DrivingSection.tsx]: commnet

File: components/WhatDrivesUs/WhatDrivesUs.tsx [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+    <Content
+      pt={12}
+      pb={{ xs: 11, md: 16 }}
+      sx={{ backgroundColor: theme.palette.common.black }}
+    >
+      <Grid
+        item
+        container
+        direction="column"
+        alignItems={{ xs: "start", md: "center" }}
```
Comment (adiusz) [inline components/WhatDrivesUs/WhatDrivesUs.tsx:23]: personally I don't like that styling as propses, I think it creates a mess, but if you all want to do it that way so let's keep it

File: components/WhatDrivesUs/WhatDrivesUs.tsx [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+        <LinearGradient shade="light" />
+      </Grid>
+      <Grid
+        container
+        direction={{ xs: "column", md: "row" }}
+        rowSpacing={{ xs: 8, md: 0 }}
+        columnSpacing={{ md: 8 }}
+      >
+        <Grid item flex={1}>
+          <DrivingSection title="Knowledge" icon="bulb">
```
Comment (adiusz) [inline components/WhatDrivesUs/WhatDrivesUs.tsx]: again perfect usecase for map method

File: components/JoinCommunityButton/JoinCommunityButton.styled.tsx [hunk 9]
```diff
@@ -21,3 +22,11 @@ export const StyledJoinCommunityButton = styled(Button)`
     color: ${({ theme }) => theme.palette.primary.main};
   }
 `;
+
+export const StyledCustomLink = styled(CustomLink)`
+  width: 205px;
```
Comment (adiusz) [inline components/JoinCommunityButton/JoinCommunityButton.styled.tsx]: fixed width? most of the time its a bad idea. Safer to use min/max width, depending on the usecase

File: components/JoinCommunityButton/JoinCommunityButton.tsx [hunk 10]
```diff
@@ -1,12 +1,15 @@
 import React from "react";
 
 import { CustomLink } from "../CustomLink";
-import { StyledJoinCommunityButton } from "./JoinCommunityButton.styled";
+import {
+  StyledCustomLink,
+  StyledJoinCommunityButton,
+} from "./JoinCommunityButton.styled";
 
 export const JoinCommunityButton = () => {
```
Comment (adiusz) [inline components/JoinCommunityButton/JoinCommunityButton.tsx]: can't we just have a more generic component like Button & ButtonOutline? that way you would just pass the children, in this case "Join The community" in some span and we can reuse it. Or would you like to create component for each button on the page? 😁

**Review by adiusz — CHANGES_REQUESTED** [review]

File: components/JoinCommunityButton/JoinCommunityButton.tsx (hunk 10 above)
Comment (TomaszKotowski) [reply components/JoinCommunityButton/JoinCommunityButton.tsx]: Changed to Button

File: components/JoinCommunityButton/JoinCommunityButton.styled.tsx (hunk 9 above)
Comment (TomaszKotowski) [reply components/JoinCommunityButton/JoinCommunityButton.styled.tsx]: On the design the with is fixed

File: components/WhatDrivesUs/WhatDrivesUs.tsx (hunk 8 above)
Comment (TomaszKotowski) [reply components/WhatDrivesUs/WhatDrivesUs.tsx]: Done

File: components/DrivingSection/DrivingSection.tsx (hunk 6 above)
Comment (TomaszKotowski) [reply components/DrivingSection/DrivingSection.tsx]: Removed

File: components/About/SingleCard.styled.tsx (hunk 5 above)
Comment (TomaszKotowski) [reply components/About/SingleCard.styled.tsx]: Removed

File: components/About/About.tsx (hunk 4 above)
Comment (TomaszKotowski) [reply components/About/About.tsx]: Done

File: components/About/SingleCard.styled.tsx (hunk 3 above)
Comment (TomaszKotowski) [reply components/About/SingleCard.styled.tsx]: Removed

File: theme/theme.ts (hunk 2 above)
Comment (TomaszKotowski) [reply theme/theme.ts]: Removed

File: theme/theme.d.ts (hunk 1 above)
Comment (TomaszKotowski) [reply theme/theme.d.ts]: Removed

**Review by 99mac99 — APPROVED** [review]

**Pull request merged**